ci/jenkins/pipelines/skuba-nightly.Jenkinsfile

```diff
@@ -18,33 +16,36 @@ pipeline {
    stages {
         stage('Getting Ready For Cluster Deployment') { steps {
             sh(script: 'make -f skuba/ci/Makefile pre_deployment', label: 'Pre Deployment')
-            sh(script: 'cd skuba; make install; cd ../', label: 'Install skuba')
+            sh(script: "pushd skuba; make -f Makefile install; popd", label: 'Build Skuba')
         } }
 
-        stage('Cluster Deployment') { steps {
-            sh(script: 'make -f skuba/ci/Makefile deploy', label: 'Deploy')
-            archiveArtifacts("skuba/ci/infra/${PLATFORM}/terraform.tfstate")
-            archiveArtifacts("skuba/ci/infra/${PLATFORM}/terraform.tfvars.json")
-        } }
+       stage('Run Integration tests') { steps {
+           sh(script: "make -f skuba/ci/Makefile test_integration", label: "test_integration")
+       } }
 
-       stage('Run end-to-end tests') { steps {
-           dir("skuba") {
-             sh(script: 'make build-ginkgo', label: 'Build ginkgo binary')
-             sh(script: "make setup-ssh", label: "Setup SSH")
-             sh(script: "make test-e2e", label: "End-to-end tests")
-       } } }
- 
    }
    post {
-       always {
-           sh(script: 'make --keep-going -f skuba/ci/Makefile post_run', label: 'Post Run')
-           zip(archive: true, dir: 'testrunner_logs', zipFile: 'testrunner_logs.zip')
-       }
-       cleanup {
-           dir("${WORKSPACE}") {
-               deleteDir()
-           }
-           sh(script: "rm -f ${SKUBA_BIN_PATH}; ", label: 'Remove built skuba')
+        always {
+            archiveArtifacts(artifacts: "skuba/ci/infra/${PLATFORM}/terraform.tfstate", allowEmptyArchive: true)
+            archiveArtifacts(artifacts: "skuba/ci/infra/${PLATFORM}/terraform.tfvars.json", allowEmptyArchive: true)
+            archiveArtifacts(artifacts: 'testrunner_logs/**/*', allowEmptyArchive: true)
```

Comment:
same here we should also collect testrunner.log file

Reply:
done